modules/images/regenerate-existing-images/load.php

```diff
@@ -77,3 +87,56 @@ function perflab_get_background_job_capabilities() {
 		'assign_terms' => 'assign_jobs', // To assign background_job terms to posts.
 	);
 }
+
+/**
+ * Creates a background job.
+ *
+ * Job refers to the term and queue refers to the `background_job` taxonomy.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $name Name of job identifier.
+ * @param array  $data Data for the job.
+ * @return Perflab_Background_Job|WP_Error Job object if created successfully, else WP_Error.
+ */
+function perflab_create_background_job( $name, array $data = array() ) {
```

Comment:
@ankitrox Okay, let's keep this function as part of this PR, and we can close #518. Can you copy what #518 does and move the function to a new `helper.php` file and update the tests file too.

Make sure to add that this PR fixes #479 in the description too.

Reply:
@jjgrainger  ✅ Done